hyperstart.go

```diff
@@ -342,6 +344,23 @@ func (h *hyper) init(pod *Pod, config interface{}) (err error) {
 	// Override pod agent configuration
 	pod.config.AgentConfig = h.config
 
+	created, err := pod.isPodCreated()
+	if err != nil {
+		return err
+	}
+
+	if !created {
+		virtLog.Infof("Pod %s has not competed creation yet, adding hypervisor devices", pod.id)
```

Comment:
typo: "completed". I'd write as, `Pod %s not fully created yet, adding hypervisor devices`